api/src/main/webapp/WEB-INF/appengine-web.xml.template

```diff
@@ -13,6 +13,9 @@
     <env-var name="spring.datasource.url" value="${DB_CONNECTION_STRING}"/>
     <env-var name="spring.datasource.username" value="${WORKBENCH_DB_USER}"/>
     <env-var name="spring.datasource.password" value="${WORKBENCH_DB_PASSWORD}"/>
+    <!-- NOTE(RW-235) Keep the db connection alive. -->
```

Comment:
Looks good.

Did you look into the spring.datasource.dbcp.test-while-idle=true settings shown in later responses? (We might have an issue with idle connections like we had in the RDR without this.)

Reply:
I couldn't tell if test-on-borrow would get a new, functional connection if it found an issue. If it does, then it should be sufficient; if not, then test-on-borrow doesn't really seeem useful, and maybe we would only need the test-while-idle.

Looking over https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/model/database.py I think we're doing some of all of these: Pinging idle connections; opening new connections ourselves if idle connections are dead; and telling SQLAlchemy to throw away old connections.

I guess I'm inclined to wait and see if Spring can do it in one, or if it too needs a variety of connection management options.

Reply:
Sounds fine. 

You might want to add a comment here with a link to the SO post, mentioning the existence of this other setting.